### PR TITLE
Add steps to get lightning goroutine

### PR DESCRIPTION
**Pull request by lance6716**
### What is changed, added or deleted? (Required)

 update FAQ from ONCALL

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by glorv** [comment]
LGTM

**Review by kennytm — APPROVED** [review]

File: tidb-lightning/tidb-lightning-faq.md [hunk 1]
```diff
@@ -355,3 +355,19 @@ TiDB Lightning Local-backend 只支持导入到 v4.0.0 及以上版本的 TiDB 
 [mydumper.csv]
 header = false
 ```
+
+## 如何获取 TiDB Lightning 运行时的 goroutine 信息
+
+1. 如果配置文件中已经指定了 [status-port](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-全局配置)，可以跳过此步骤。否则，需要向 TiDB Lightning 发送 USR1 信号开启 status-port。
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-faq.md:361]:
```suggestion
1. 如果 TiDB Lightning 的配置文件中已经指定了 [`status-port`](/tidb-lightning/tidb-lightning-configuration.md#tidb-lightning-全局配置)，可以跳过此步骤。否则，需要向 TiDB Lightning 发送 USR1 信号以开启 `status-port`。
```

File: tidb-lightning/tidb-lightning-faq.md [hunk 2]
```diff
@@ -355,3 +355,19 @@ TiDB Lightning Local-backend 只支持导入到 v4.0.0 及以上版本的 TiDB 
...
+
+   首先通过 `ps` 等命令获取 TiDB Lightning 的进程 ID（PID），然后运行如下命令
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-faq.md:363]: 能否给出完整的 `PS` 命令呢？
```suggestion
    首先通过 `ps` 等命令获取 TiDB Lightning 的进程 PID，然后运行如下命令：
```

File: tidb-lightning/tidb-lightning-faq.md [hunk 3]
```diff
@@ -355,3 +355,19 @@ TiDB Lightning Local-backend 只支持导入到 v4.0.0 及以上版本的 TiDB 
...
+
+   {{< copyable "shell-regular" >}}
+
+   ```sh
+   kill -USR1 <lightning-pid>
+   ```
+   
+   查看 TiDB Lightning 的日志，其中有“starting HTTP server / start HTTP server / started HTTP server”的日志显示了新开启的 status-port。
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-faq.md:371]:
```suggestion
    {{< copyable "shell-regular" >}}

    ```sh
    kill -USR1 <lightning-pid>
    ```
   
    查看 TiDB Lightning 的日志，其中 `starting HTTP server` / `start HTTP server` / `started HTTP server` 的日志会显示新开启的 `status-port`。
```

File: tidb-lightning/tidb-lightning-faq.md [hunk 4]
```diff
@@ -355,3 +355,19 @@ TiDB Lightning Local-backend 只支持导入到 v4.0.0 及以上版本的 TiDB 
...
+
+2. 访问 `http://<lightning-ip>:<status-port>/debug/pprof/goroutine?debug=2` 返回 goroutine 信息。
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-faq.md:373]:
```suggestion
2. 访问 `http://<lightning-ip>:<status-port>/debug/pprof/goroutine?debug=2` 可获取 goroutine 信息。
```

**Comment by lance6716** [comment]
CI error:

functions-and-operators/expressions-pushed-down.md: this file has manual line breaks in the following lines:

🤔

File: tidb-lightning/tidb-lightning-faq.md (hunk 2 above)
Comment (lance6716) [reply tidb-lightning/tidb-lightning-faq.md:363]: ps 在不同发行版的命令不一样 `ps -ef`、`ps aux`，不太好统一给一个。而且这个命令很常用，相信用户自己会的

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**